apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/chat/group/GroupChatInfoView.kt

```diff
@@ -314,7 +337,7 @@ fun ModalData.GroupChatInfoLayout(
       if (s.isEmpty()) members else members.filter { m -> m.anyNameContains(s) }
     }
   }
-  Box {
+  Box(Modifier.alpha(if (disabled.value) 0.6f else 1f)) {
```

Comment:
what purpose of introducing another way of showing progress here? There was no such thing in the app before